mozilla/bedrock
# Update platform templates with trailhead look and feel

## Description

- separate download and thanks templates & pages into trailhead and protocol variants 
    - based on which lang file they use
- separate styles and assets into trailhead and quantum variants 
    - based on design appearance
- update protocol template to use trailhead appearance
- add navigation menu to platform pages
- update feature "icons" from quantum design to have trailhead colours
    - and optimize
- simplify protocol template:
    - remove out of date browser notification
    - remove newsletter signup up option from protocol thanks page
    - remove fxa cta from protocol download page
    - the pages using these templates have old translations so do not display this content anyway
- move asset only used by feature pages into media/images/quantum
- updated Yandex feature images to be trailhead coloured 

## Issue / Bugzilla link

Fix #8532

## Testing

- enable `dev=False` in .env
- enable `SWITCH_FIREFOX_YANDEX` in .env
- make sure locale files are up to date (run `./manage.py l10n_update` in the venv)
- all tempaltes: "other platforms" still opens modal
- trailhead in Firefox but logged out: still prompted to create an account when click download

---

- http://localhost:8000/an/firefox/new/ (protocol template)
- http://localhost:8000/en-US/firefox/linux/ (protocol template)
- http://localhost:8000/en-US/firefox/mac/ (protocol template)
- http://localhost:8000/en-US/firefox/windows/ (protocol template)
- http://localhost:8000/en-US/firefox/new/ (trailhead template)
- http://localhost:8000/en-CA/firefox/new/ (trailhead template WITH special title attribute)
- http://localhost:8000/en-US/firefox/new/?geo=ru (trailhead template)
- http://localhost:8000/ru/firefox/new/?geo=ru (yandex template)
